src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py

```diff
@@ -896,6 +920,18 @@ def __call__(
                 if self.do_classifier_free_guidance:
                     noise_pred_uncond, noise_pred_text = noise_pred.chunk(2)
                     noise_pred = noise_pred_uncond + self.guidance_scale * (noise_pred_text - noise_pred_uncond)
+                    should_skip_layers = True if i > num_inference_steps * skip_layer_guidance_start and i < num_inference_steps* skip_layer_guidance_stop else False
+                    if skip_guidance_layers is not None and should_skip_layers:
```

Comment:
so `if skip_guidance_layers is None`, which is the current default, we do not run the model?

Reply:
right, the noise pred and uncond are already calculated. this is for SLG.

Reply:
ohh got it, thanks!